src/ssl.c

```diff
@@ -38678,17 +38828,19 @@ int wolfSSL_X509_NAME_print_ex(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
             }
             totalSz += tmpSz;
         }
+        if (totalSz > 0 && fullName[totalSz-1] == '\0')
```

Comment:
Why is this necessary?
Is this something that can be fixed above?

Reply:
As per my comment above the function:
> The BIO output of  wolfSSL_X509_NAME_print_ex does NOT include the null terminator
The null terminator is automatically appended by both `XSNPRINTF` and `XSTRNCAT`. Both also require C strings as input so removing the null terminator at that stage is not possible. The simplest solution I found is decrementing the `totalSz` at the end.

Reply:
Actually nevermind. I found a way.